src/TestFramework/Codeception/Config/Builder/InitialConfigBuilder.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * Copyright © 2018 Tobias Stadler
+ *
+ * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Infection\TestFramework\Codeception\Config\Builder;
+
+use Infection\TestFramework\Config\InitialConfigBuilder as ConfigBuilder;
+
+class InitialConfigBuilder implements ConfigBuilder
+{
+    /**
+     * @var string
+     */
+    private $originalConfigPath;
+
+    public function __construct(string $originalConfigPath)
+    {
+        $this->originalConfigPath = $originalConfigPath;
+    }
+
+    public function build(): string
+    {
+        return $this->originalConfigPath;
```

Comment:
Maybe it's better to create all needed for Infection parameters in a new, temp config like it is done for PHPUnit and PHPSpec rather that adding/overriding config in `ArgumentsAndOptionsBuilder`?

This will probably help to avoid passing `Mutant $mutant` parameter to `ArgumentsAndOptionsBuilder`

In other words, I suggest to move this code:

```php
$options[] = '-o "paths: output: ' . $this->tempDir . '"';
$options[] = '-o "coverage: enabled: true"';
$options[] = '--coverage-phpunit ' . CodeCoverageData::CODECEPTION_COVERAGE_DIR;
```

from `ArgumentsAndOptionsBuilder` and create temp config. It will be consistent with the other Test Framework and will help to avoid changing the interface

Reply:
unfortunately Codeception does not handle absolute paths in its config files, but maybe I can get it working using relative paths.

Reply:
I'd suggest creating Codeception's Extension which will transform the current config the way you need it. 
Extension can be published as a standalone package